api/src/main/java/org/openmrs/util/HandlerUtil.java

```diff
@@ -78,8 +78,11 @@ public boolean equals(Object obj) {
 			}
 			if (type == null) {
 				return other.type == null;
-			} else
+			}
```

Comment:
Thanks @miirochristopher for working on this, could you remove some spaces, Also try to  do identation